unit_of_time/__init__.py

```
@@ -14,42 +18,157 @@ def date_from_int(val, div=1):
+        return cls.get_index_for_date(cls.last_date) + 1
+
+    def __int__(self):
+        """
+        Provide the integer identifier for this time unit kind.
+
+        Returns:
+            int: The kind's integer identifier.
+        """
+        return self.kind_int
+
     def __int__(cls):
```

Ruff (0.13.3)

Redefinition of unused `__int__` from line 163

(F811)